stories/cloudprogressbar.js

```diff
@@ -0,0 +1,57 @@
+//@flow
+import React from "react";
+import CloudProgressBar from "../src/lib/components/cloudprogressbar/CloudProgressBar.component";
+import { Wrapper, Title, SubTitle } from "./common";
+import { storiesOf } from "@storybook/react";
+
+storiesOf("CloudProgressBar", module).add("Default", () => {
+  return (
+    <Wrapper>
+      <Title>CloudProgressBar</Title>
+      <div style={{ width: "200px", margin: '20px 0 20px 0' }}>
+        <CloudProgressBar
+          percentage={30}
+          borderSize='0.5px'
+          cloudColor='yellow'
+          progressCloudColor='blue'
+        >
+          <Title>30%</Title>
+          <SubTitle>of the quotas has been reached.</SubTitle>
+        </CloudProgressBar>
+      </div>
+      <div style={{ width: "300px", margin: '20px 0 20px 0' }}>
+        <CloudProgressBar
+          percentage={50}
+          borderSize='0.5px'
+          cloudColor='#ccc'
+          progressCloudColor='#000'
+        >
+          <Title>50%</Title>
+          <SubTitle>of the quotas has been reached</SubTitle>
+        </CloudProgressBar>
+      </div>
```

Comment:
Maybe I didn't make it clear in my last review. 
We still want to display the text `50% of the quotas has been reached` inside the cloud. So basically we will want to give a style to the text.

```     
    <div style={{ width: "300px", margin: "20px 0 20px 0" }}>
        <CloudProgressBar
          percentage={50}
          borderSize="0.5px"
          cloudColor="#ccc"
          progressCloudColor="#000"
        >
          <div
            style={{
              position: "absolute",
              top: "50%",
              left: "50%",
              transform: "translate(-50%,-50%)"
            }}
          >
            <Title>50%</Title>
            <SubTitle>of the quotas has been reached</SubTitle>
          </div>
        </CloudProgressBar>
      </div>
```